server/app.mjs

```diff
@@ -327,17 +330,6 @@ const mount = async (
     for (let block of script) {
       if (block.type !== 'text') {
         block.name = newName || block.name;
-        block.tools = [
```

Comment:
@thedadams We actually change it back to be a remote tool so it is easy to work with. So no longer need to do special logic when copying now. Sorry about that.